Add link to airflow metrics (#9425)
DataDog/integrations-core: airflow/README.md

##### Connect Airflow to DogStatsD

> Connect Airflow to DogStatsD (included in the Datadog Agent) by using the Airflow `statsd` feature to collect metrics. For more information about the metrics reported by the Airflow version used and the extra option available in Airflow, please see the [documentation][16].

Reviewer: ```suggestion
Connect Airflow to DogStatsD (included in the Datadog Agent) by using the Airflow `statsd` feature to collect metrics. For more information about the metrics reported by the Airflow version that is being used, and about the additional options available in Airflow, see the [Airflow documentation][16].
```
